code/modules/pai/candidate.dm

```diff
@@ -0,0 +1,30 @@
+/// Created when a user clicks the "pAI candidate" window
+/datum/pai_candidate
+	/// User inputted OOC comments
+	var/comments
+	/// User inputted behavior description
+	var/description
+	/// User's ckey
+	var/ckey
+	/// User's pAI name. If blank, ninja name.
+	var/name
+	/// If the user has hit "submit"
+	var/ready = FALSE
+
+/datum/pai_candidate/New(mob/user)
+	src.ckey = user.ckey
```

Comment:
would it not make more sense to pass in the ckey directly here since the mob is usually just a ghost?